doxyqml/doxyqml.py

```diff
@@ -78,7 +80,7 @@ def find_qmldir_file(qml_file):
         dir = parent
 
 
-def find_classname(qml_file):
+def find_classname(qml_file, namespace=[]):
```

Comment:
It's not a good idea to use arrays (or any mutable type) as default argument values because if the function modifies the argument it modifies the default value. It is not the case here, but to be safe it's better to turn this into `namespace=None`.

Reply:
Ya, good point. Fixed.